src/app/stores/defaults.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+import { Dataset, Specification, uniqueID } from "../../core";
+
+export function createDefaultGlyph(tableName: string) {
+  return {
+    _id: uniqueID(),
+    classID: "glyph.rectangle",
+    properties: { name: "Glyph" },
+    table: tableName,
+    marks: [
+      {
+        _id: uniqueID(),
+        classID: "mark.anchor",
+        properties: { name: "Anchor" },
+        mappings: {
+          x: {
+            type: "parent",
+            parentAttribute: "icx"
+          } as Specification.ParentMapping,
+          y: {
+            type: "parent",
+            parentAttribute: "icy"
+          } as Specification.ParentMapping
+        }
+      }
+    ],
+    mappings: {},
+    constraints: []
+  } as Specification.Glyph;
+}
+
+export function createDefaultChart(dataset: Dataset.Dataset) {
+  const tableName = dataset.tables[0].name;
+  const rows = dataset.tables[0].rows;
+  const glyph = createDefaultGlyph(tableName);
+  return {
+    _id: uniqueID(),
+    classID: "chart.rectangle",
+    properties: {
+      name: "Chart",
+      backgroundColor: null,
+      backgroundOpacity: 1
+    },
+    mappings: {
+      marginTop: { type: "value", value: 80 } as Specification.ValueMapping
+    },
+    glyphs: [glyph],
+    elements: [
+      {
```

Comment:
These two (the plot segment and the title) would be nice as function calls, similar to how you did createDefaultGlyph..

```
elements: [
    createDefaultPlotSegment(),
    createDefaultTitle()
]
```